ECIPs/ECIP-1053.md

```diff
@@ -0,0 +1,64 @@
+# Add OpenRPC Service Discovery To JSON-RPC Services
+
+    ECIP: 1053
+    Title: Add OpenRPC Service Discovery To JSON-RPC Services
+    Author: Shane Jonas, Zachary Belford
+    Status: DRAFT
```

Comment:
tiny nitpick that can be ignored., but can we make `DRAFT` to be `Draft`

Reply:
```suggestion
    Status: Draft
```